spring-integration-sftp/src/test/java/org/springframework/integration/sftp/dsl/SftpTests.java

```diff
@@ -117,12 +119,14 @@ public void testSftpInboundStreamFlow() throws Exception {
 		assertNotNull(message);
 		assertThat(message.getPayload(), instanceOf(InputStream.class));
 		assertThat(message.getHeaders().get(FileHeaders.REMOTE_FILE), isOneOf(" sftpSource1.txt", "sftpSource2.txt"));
+		StreamUtils.copy((InputStream) message.getPayload(), new ByteArrayOutputStream());
```

Comment:
Well, actually it does, if we check the result of the `File.delete()`.
And that is exactly what we are doing now with all those your new log diagnostics.

OK, merging having your agreement...
I also will increase the timeout in that Redis test